distribution/changelog.txt

```diff
@@ -5,10 +5,12 @@
 - Feature: [#7694] Debug option to visualize paths that the game detects as wide.
 - Feature: [#7713] The virtual floor now takes land ownership rights into account.
 - Feature: [#7771] Danish translation.
+- Feature: [#7816] Slow game speed step.
```

Comment:
Change to "[#7816, #7831] Slow game speed step.", move to end of list, and remove the duplicate lines